debian/copyright

```diff
@@ -0,0 +1,23 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: neofs-node
+Upstream-Contact: tech@nspcc.ru
+Source: https://github.com/nspcc-dev/neofs-node
+
+Files: *
+Copyright: 2018-2022 NeoSPCC (@nspcc-dev), contributors of neofs-node project 
+           (https://github.com/nspcc-dev/neofs-node/blob/master/CREDITS.md)
+
+License:
```

Comment:
IIRC there should be a short license identifier on this line, `GPL-3+`.

Reply:
it's not "any" golang I afraid it needs and too many systems have it installed manually :-(
dh-golang's not used 
idenifier inserted, thanks for correction